test/jest/utils.spec.ts

```diff
@@ -146,6 +146,27 @@ describe('Utils', () => {
 
                 expect(queryStringParser(url, [])).toEqual(expectedResult);
             });
+
+            it('should handle non-standard characters or malformed urls', () => {
```

Comment:
I ran this test against the previous code in src/utils and it passes.  I would expect the sample to not pass, but maybe I'm doing something wrong?

Reply:
using `a%AFc` as a value should cause the previous code to fail, but current code to pass.